packages/client/hmi-client/src/components/model/petrinet/model-configurations/tera-stratified-matrix.vue

```diff
@@ -109,16 +109,17 @@ const props = defineProps<{
 	id: string;
 	stratifiedMatrixType: StratifiedMatrix;
 	shouldEval: boolean;
+	matrixType?: string;
 }>();
 
 const emit = defineEmits(['update-cell-value']);
 
 const matrixTypes = ['subjectOutcome', 'subjectControllers', 'outcomeControllers', 'other'];
-const matrixType = ref('subjectOutcome');
+const matrixType = ref(props.matrixType || 'subjectOutcome');
 const matrixMap = ref<any>();
 const matrix = ref<any>([]);
 
-const currentMatrixtype = ref('');
```

Comment:
What does `currentMatrixType` do differently from `matrixType`?

Reply:
If I recall, this was introduced because of the refreshing problems in updating model-configuration, I needed an extra variable to keep track across component-refreshes.